tests/test_stainaugment.py

```diff
@@ -0,0 +1,83 @@
+"""Tests for stain augmentation code."""
+
+import pathlib
+
+import albumentations as alb
+import numpy as np
+import pytest
+
+from tiatoolbox.data import stainnorm_target
+from tiatoolbox.tools.stainaugment import StainAugmentation
+from tiatoolbox.tools.stainnorm import get_normaliser
+from tiatoolbox.utils.misc import imread
+
+
+def test_stainaugment(source_image, norm_vahadane):
```

Comment:
If the function is testing with Vahadane only, why can't we name it as test_vahadane_stainaugment() ?

Reply:
Both Vahadane and Macenko methods are tested in the function, however, only files `norm_vahadane` and `source_image` are used in some part of the test.